src/app/components/question-management/question-management.component.html

```diff
@@ -0,0 +1,36 @@
+<!--<pre>{{ questionCards$ | async | json}}</pre>-->
+<div class="wrapper">
+  <div *ngIf="questionCards$ | async as questionsInfo;">
```

Comment:
; is not needed

Reply:
better to call var same as observables (more clear)
`questionCards$ | async as questionCards`